parts/linux/cloud-init/artifacts/components.json

```diff
@@ -27,7 +27,8 @@
       "amd64OnlyVersions": [],
       "multiArchVersions": [
         "v1.9.4-hotfix.20240520",
```

Comment:
will we be able to remove the oldest image sometime soon?

Reply:
yes, I have been tracking the previous rollout. Once all the clusters are on "v1.9.4-hotfix.20240627", I will remove "v1.9.4-hotfix.20240520". But currently, there are 1000s of clusters still on "v1.9.4-hotfix.20240520".